scripts/manual_operations/manual_submission.py

```diff
@@ -87,12 +87,30 @@ def get_location_and_rb_from_icat(icat_client, instrument, run_number, file_ext)
         print("ICAT not connected")  # pragma: no cover
         sys.exit(1)  # pragma: no cover
 
-    icat_instrument_prefix = get_icat_instrument_prefix(instrument)
-    file_name = f"{icat_instrument_prefix}{str(run_number).zfill(5)}.{file_ext}"
+    # look for file-name assuming file-name uses full instrument name
+    file_name = f"{instrument}{str(run_number).zfill(5)}.{file_ext}"
     datafile = icat_client.execute_query("SELECT df FROM Datafile df WHERE df.name = '"
                                          + file_name +
                                          "' INCLUDE df.dataset AS ds, ds.investigation")
```

Comment:
Given we use this same query 4 times by my count, I wonder if it'd be worth refactoring.
The first idea that comes to mind would be a simple method:
```python
def create_query(file_name):
    return f"SELECT df FROM Datafile df WHERE df.name = '{file_name}' INCLUDE df.dataset AS ds, ds.investigation"
```

Reply:
thanks and implemented and with a few smaller updates to tests also